### PR TITLE
Scheduler Fix for Info section

### PR DESCRIPTION
**Pull request by jayasimha-raghavan-unskript**
## Description
Please include a summary of the change, motivation and context.
* Fixed parsing of info section
* Fixed the issue of insertion of the --info flag.

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

```
###################################
Processing global section
###################################
Global: audit period 90 days
###################################
Processing checks section
###################################
###################################
Processing credential section
###################################
Credential: Skipping type aws, name awscreds
Credential: Skipping type k8s, name k8screds
Credential: Skipping type gcp, name gcpcreds
Credential: Skipping type elasticsearch, name escreds
Credential: Skipping type redis, name rediscreds
Credential: Skipping type postgres, name postgrescreds
Credential: Skipping type mongodb, name mongodbcreds
Credential: Skipping type kafka, name kafkacreds
Credential: Skipping type rest, name restcreds
Credential: Skipping type vault, name vaultcreds
Credential: Skipping type keycloak, name keycloakcreds
###################################
Processing jobs section
###################################
Job: test contains connector types: k8s
###################################
Processing remote debugging section
###################################
Remote_debugging: No remote_debugging config found
###################################
Processing scheduler section
###################################
Schedule: cadence */60 * * * *, job name: test
Schedule: Programming crontab */60 * * * * /usr/local/bin/unskript-ctl.sh run check --type k8s --info --report
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py

```

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

**Comment by amit-chandak-unskript** [comment]
Please test out the following combinations @jayasimha-raghavan-unskript 

- only info
- only checks with type
- only checks with name
- only scripts
- total: 4C1 + 4C2 + 4C3 + 4C4 = 4 + 12 + 4 + 1 = 21 combinations

**Comment by jayasimha-raghavan-unskript** [comment]
### Only Info
```
root@awesome-awesome-runbooks-0:/# unskript-ctl.sh run --info


Information Gathering Action Results
Running: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:23<00:00, 23.44s/it]

k8s/k8s_get_service_images


Using incluster config
No labels found for service kubelet in namespace lightbeam. Skipping...
+------------------------------------------------------+--------------------------------------------------+
| Service (Namespace)                                  | Images                                           |
+======================================================+==================================================+
| kafka-ui (lightbeam)                                 | docker.io/fastcomply/kafka-ui                    |
+------------------------------------------------------+--------------------------------------------------+
| kong-proxy (lightbeam)                               | docker.io/fastcomply/kong-oidc                   |
|                                                      | fastcomply/kong-ingress-controller               |
+------------------------------------------------------+--------------------------------------------------+
| kong-validation-webhook (lightbeam)                  | docker.io/fastcomply/kong-oidc                   |
|                                                      | fastcomply/kong-ingress-controller               |
+------------------------------------------------------+--------------------------------------------------+
| lb-argo-workflows-server (lightbeam)                 | docker.io/bitnami/argo-workflow-cli              |
+------------------------------------------------------+--------------------------------------------------+
| lb-kafka-connect (lightbeam)                         | docker.io/fastcomply/lightbeam-kafka-connect     |
+------------------------------------------------------+--------------------------------------------------+
| lb-keycloak (lightbeam)                              | docker.io/fastcomply/lb-keycloak                 |
+------------------------------------------------------+--------------------------------------------------+
| lb-keycloak-headless (lightbeam)                     | docker.io/fastcomply/lb-keycloak                 |
+------------------------------------------------------+--------------------------------------------------+
| lb-redis-headless (lightbeam)                        | docker.io/fastcomply/redis                       |
+------------------------------------------------------+--------------------------------------------------+
| lb-redis-master (lightbeam)                          | docker.io/fastcomply/redis                       |
+------------------------------------------------------+--------------------------------------------------+
| lb-redis-replicas (lightbeam)                        | docker.io/fastcomply/redis                       |
+------------------------------------------------------+--------------------------------------------------+
| lb-vault (lightbeam)                                 | docker.io/fastcomply/vault-unseal                |
|                                                      | docker.io/fastcomply/vault                       |
+------------------------------------------------------+--------------------------------------------------+
| lb-vault-ui (lightbeam)                              | docker.io/fastcomply/vault-unseal                |
|                                                      | docker.io/fastcomply/vault                       |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-api-gateway (lightbeam)                    | docker.io/fastcomply/api-gateway                 |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-datahub-gms (lightbeam)                    | docker.io/fastcomply/datahub-gms                 |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-datahub-gms-graphql (lightbeam)            | docker.io/fastcomply/datahub-gms-graphql-service |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-elasticsearch-master (lightbeam)           | docker.io/fastcomply/lightbeam-elasticsearch     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-elasticsearch-master-headless (lightbeam)  | docker.io/fastcomply/lightbeam-elasticsearch     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-elasticsearch-nodeport-service (lightbeam) | docker.io/fastcomply/lightbeam-elasticsearch     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-files-service (lightbeam)                  | docker.io/fastcomply/rex                         |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-frontend (lightbeam)                       | docker.io/fastcomply/frontend                    |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka (lightbeam)                          | docker.io/fastcomply/kafka                       |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka-headless (lightbeam)                 | docker.io/fastcomply/kafka                       |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka-metrics (lightbeam)                  | docker.io/fastcomply/kafka-exporter              |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka-zookeeper (lightbeam)                | docker.io/fastcomply/zookeeper                   |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka-zookeeper-headless (lightbeam)       | docker.io/fastcomply/zookeeper                   |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-kafka-zookeeper-metrics (lightbeam)        | docker.io/fastcomply/zookeeper                   |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-minio-svc (lightbeam)                      | docker.io/fastcomply/minio                       |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-mongodb-arbiter-headless (lightbeam)       | docker.io/fastcomply/mongodb                     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-mongodb-headless (lightbeam)               | docker.io/fastcomply/mongodb                     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-mongodb-nodeport-service (lightbeam)       | docker.io/fastcomply/mongodb                     |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-policy-engine (lightbeam)                  | docker.io/fastcomply/policy-engine               |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-presidio-server (lightbeam)                | docker.io/fastcomply/presidio-server             |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-prometheus (lightbeam)                     | docker.io/fastcomply/prometheus                  |
|                                                      | docker.io/fastcomply/prometheus-config-reloader  |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-prometheus-pushgateway (lightbeam)         | docker.io/fastcomply/pushgateway                 |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-qdrant (lightbeam)                         | docker.io/fastcomply/qdrant                      |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-qdrant-headless (lightbeam)                | docker.io/fastcomply/qdrant                      |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-report-engine (lightbeam)                  | docker.io/fastcomply/report-engine               |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-schema-registry (lightbeam)                | docker.io/fastcomply/cp-schema-registry          |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-text-extraction (lightbeam)                | docker.io/fastcomply/text-extraction             |
|                                                      | docker.io/gotenberg/gotenberg                    |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-text-extraction-headless (lightbeam)       | docker.io/fastcomply/text-extraction             |
|                                                      | docker.io/gotenberg/gotenberg                    |
+------------------------------------------------------+--------------------------------------------------+
| lightbeam-tika-server (lightbeam)                    | docker.io/fastcomply/tika-server                 |
+------------------------------------------------------+--------------------------------------------------+
| postgres (lightbeam)                                 | docker.io/fastcomply/bitnami-postgres-exporter   |
|                                                      | docker.io/fastcomply/bitnami-postgresql          |
+------------------------------------------------------+--------------------------------------------------+
| postgres-metrics (lightbeam)                         | docker.io/fastcomply/bitnami-postgres-exporter   |
|                                                      | docker.io/fastcomply/bitnami-postgresql          |
+------------------------------------------------------+--------------------------------------------------+
| prometheus-operated (lightbeam)                      | docker.io/fastcomply/prometheus                  |
|                                                      | docker.io/fastcomply/prometheus-config-reloader  |
+------------------------------------------------------+--------------------------------------------------+
| vault-agent-injector-svc (lightbeam)                 | docker.io/fastcomply/vault-k8s                   |
+------------------------------------------------------+--------------------------------------------------+

###
root@awesome-awesome-runbooks-0:/# 
```
 
### Only Checks with Type

```
root@awesome-awesome-runbooks-0:/# unskript-ctl.sh run check --type k8s
Running: 100%|██████████████████████████████████████████████████████| 62/62 [02:52<00:00,  2.79s/it]

╒════════════════════════════════════════════════════════╤══════════╤════════════════╤═════════╕
│ Checks Name                                            │ Result   │   Failed Count │ Error   │
╞════════════════════════════════════════════════════════╪══════════╪════════════════╪═════════╡
│ Check K8s worker CPU Utilization                       │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get expiring K8s certificates                          │  FAIL    │              2 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8S Service with no associated endpoints           │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes PODS with high restart                  │  FAIL    │              2 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Check the status of K8s CronJob pods                   │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Detect K8s service crashes                             │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Check K8s services endpoint and SSL certificate health │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes Nodes that have insufficient resources  │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get all K8s Pods in ImagePullBackOff State             │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get all K8s Pods in CrashLoopBackOff State             │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Deployment Status                                  │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes Error PODs from All Jobs                │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8s offline nodes                                  │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get frequently restarting K8s pods                     │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes PODs in not Running State               │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8s services exceeding memory utilization          │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes Unbound PVCs                            │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get all K8s Pods in Terminating State                  │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Check K8s service PVC utilization                      │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8S OOMKilled Pods                                 │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get All Evicted PODS From Namespace                    │  PASS    │              0 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8S Cluster Health                                 │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get K8s get pending pods                               │  FAIL    │              1 │ N/A     │
├────────────────────────────────────────────────────────┼──────────┼────────────────┼─────────┤
│ Get Kubernetes Failed Deployments                      │  PASS    │              0 │ N/A     │
╘════════════════════════════════════════════════════════╧══════════╧════════════════╧═════════╛

k8s:Get expiring K8s certificates
Failed Objects:
- - namespace: lightbeam
    secret_name: lb-https-secret
  - namespace: lightbeam
    secret_name: lbdemoeg
  - namespace: lightbeam
    secret_name: vault-tls-server
- - namespace: unskript
    secret_name: lbdemoeg
  - namespace: unskript
    secret_name: lbunskript
  - namespace: unskript
    secret_name: lbunskript1
  - namespace: unskript
    secret_name: letsencrypt-cert-unskript
  - namespace: unskript
    secret_name: unskript-proxy

 
k8s:Get K8S Service with no associated endpoints
Failed Objects:
- - name: lightbeam-kafka-zookeeper-metrics
    namespace: lightbeam

 
k8s:Get Kubernetes PODS with high restart
Failed Objects:
- - name: lb-vault-0
    namespace: lightbeam
- - name: tenantmgr-858f4b46bc-thdrt
    namespace: unskript

 
k8s:Check K8s services endpoint and SSL certificate health
Failed Objects:
- Error: No endpoints specified.

 
k8s:Get all K8s Pods in ImagePullBackOff State
Failed Objects:
- - container: lightbeam-policy-consumer
    namespace: lightbeam
    pod: lightbeam-policy-consumer-65d546cc4d-x6nsx

 
k8s:Get all K8s Pods in CrashLoopBackOff State
Failed Objects:
- - container: vault
    namespace: lightbeam
    pod: lb-vault-0

 
k8s:Get Deployment Status
Failed Objects:
- - deployment_name: lightbeam-policy-consumer
    namespace: lightbeam

 
k8s:Get frequently restarting K8s pods
Failed Objects:
- namespace: lightbeam
  pod: lb-vault-0

 
k8s:Get Kubernetes PODs in not Running State
Failed Objects:
- - name: kong-migrations-sfdcb
    namespace: lightbeam

 
k8s:Get K8S Cluster Health
Failed Objects:
- abnormal_nodes: []
  not_ready_nodes: []
  not_ready_pods:
  - condition_message: 'containers with unready status: [vault]'
    condition_reason: ContainersNotReady
    condition_status: 'False'
    condition_type: Ready
    name: lb-vault-0
    namespace: lightbeam
  - condition_message: 'containers with unready status: [lightbeam-policy-consumer]'
    condition_reason: ContainersNotReady
    condition_status: 'False'
    condition_type: Ready
    name: lightbeam-policy-consumer-65d546cc4d-x6nsx
    namespace: lightbeam
  - condition_message: 'containers with unready status: [mongodb]'
    condition_reason: ContainersNotReady
    condition_status: 'False'
    condition_type: Ready
    name: mongodb-0
    namespace: unskript
  - condition_message: 'containers with unready status: [jupyterlab]'
    condition_reason: ContainersNotReady
    condition_status: 'False'
    condition_type: Ready
    name: vanila-minimal-notebook-0
    namespace: vanila

 
k8s:Get K8s get pending pods
Failed Objects:
- - - lightbeam-policy-consumer-65d546cc4d-x6nsx
    - lightbeam

```

**Review by amit-chandak-unskript — COMMENTED** [review]
Don't see the fix for info section missing

**Comment by jayasimha-raghavan-unskript** [comment]
Additional Tests that were done
*  check + info 
*  info only
*  script + check + info
*  script + info only 
*  check only
*  script only

File: unskript-ctl/unskript_ctl_config_parser.py [hunk 1]
```diff
@@ -133,16 +134,13 @@ def parse(self):
                 combined_script = ';'.join(filtered_scripts)
                 print(f'Job: {self.job_name} contains custom script: {combined_script}')
                 if combine_check_types_and_script or combine_check_names_and_script:
-                    full_command += f' --script "{combined_script}" {notify}'
+                    full_command += f' --script "{combined_script}" {info} {notify}'
```
Comment (amit-chandak-unskript) [inline unskript-ctl/unskript_ctl_config_parser.py:137]: What happens, if you have check types, script and info, wont it show --info 2 times, as it got added in line 112. Can you tabulate combinations and their corresponding generated cli command in a table and put that in the PR

Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript_ctl_config_parser.py]: ## Here is the Full Table of 15 Combination. 

*  unskript-ctl.sh run --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --type <TYPE>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run --script <SCRIPT>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unksirpt-ctl.sh run --script <SCRIPT> --info
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run --script "/usr/local/bin/lb_pvc.sh"  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --name <NAME> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py

```

*  unskript-ctl.sh run check --name <NAME> --script <SCRIPT>
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --script <SCRIPT>
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```
*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes ; /usr/local/bin/unskript-ctl.sh run check --type k8s 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```


*  unskript-ctl.sh run check --type <TYPE> check --name <NAME> --script <SCRIPT>

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes ; /usr/local/bin/unskript-ctl.sh run check --type k8s  --script "/usr/local/bin/lb_pvc.sh"  
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --type <TYPE> --script <SCRIPT> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME> --script <SCRIPT> --info 
```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py

```

*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> --info  

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s  --info
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

*  unskript-ctl.sh run check --name <NAME> check --type <TYPE> --script <SCRIPT> --info 

```
Schedule: cadence 0 0 * * *, job name: lightbeam
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 
Adding audit log deletion cron job entry, 0 0 * * * /opt/conda/bin/python /usr/local/bin/unskript_audit_cleanup.py
```

Comment (amit-chandak-unskript) [reply unskript-ctl/unskript_ctl_config_parser.py]: @jayasimha-raghavan-unskript this doesnt look right to me, --info is repeated in both the commands
```
Schedule: Programming crontab 0 0 * * * /usr/local/bin/unskript-ctl.sh run check --name k8s_get_offline_nodes  --info; /usr/local/bin/unskript-ctl.sh run check --type k8s --info --script "/usr/local/bin/lb_pvc.sh" 

```
This is this case **unskript-ctl.sh run check --name check --type --script <SCRIPT> --info**

Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript_ctl_config_parser.py]: Took care of both scenarios.

Comment (jayasimha-raghavan-unskript) [reply unskript-ctl/unskript_ctl_config_parser.py]: Updated with report too

File: unskript-ctl/unskript_ctl_config_parser.py [hunk 2]
```diff
@@ -94,58 +95,65 @@ def parse(self):
...
-            cmds.append('--info')
-
+        else:
+            if info:
+                full_command = f'{UNSKRIPT_CTL_BINARY} run {info} {notify}'
+                if len(cmds) == 0:
+                    cmds.append(full_command)
+
+        info_exists = False
+        for idx,c in enumerate(cmds):
```
Comment (amit-chandak-unskript) [inline unskript-ctl/unskript_ctl_config_parser.py:147]: What is this logic doing?

Comment (amit-chandak-unskript) [reply unskript-ctl/unskript_ctl_config_parser.py:147]: @jayasimha-raghavan-unskript please take care of in your next PR

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**